bibtex-actions.el

```diff
@@ -472,17 +472,22 @@ TEMPLATE."
 
 (defun bibtex-actions-org-cite-insert (&optional multiple)
   "Return a list of keys when MULTIPLE, or else a key string."
+  ;; FIX
```

Comment:
Somewhere in here is the problem, I think.

In the wip-cite-new branch, just before merger, I added this code so the function would return a list, or an individual string.

But since mainlining, this has broken, and nothing I had tried has fixed it.